frigate/detectors/plugins/rknn.py

```diff
@@ -75,14 +75,22 @@ def get_soc(self):
 
         return soc
 
-    def parse_model_input(self, model_path, soc):
+    def parse_model_input(self, config, soc):
+        model_path = config.model.path
+
         model_props = {}
 
         # find out if user provides his own model
         # user provided models should be a path and contain a "/"
         if "/" in model_path:
             model_props["preset"] = False
             model_props["path"] = model_path
+            if config.model.model_type:
+                model_props["model_type"] = config.model.model_type
+            else:
+                raise Exception(
+                    "You must specify model_type if specifying your own model file."
+                )
```

Comment:
I know that config.model.model_type defaults to ssd so this isn't needed, but following defensive programming and separation of concern best practices, I deemed this the best way forward.